app/src/main/res/values-en/strings.xml

```diff
@@ -997,4 +997,17 @@ You will be asked to enter your main seed phrase. This is a set of **24 words**
 \n\n
 If you have any “Legacy” Accounts (created on the Olympia network) to import - or any Accounts using a Ledger hardware wallet device - please continue and then use the **Account Recovery Scan** option in your Radix Wallet settings under **Account Security**.</string>
   <string name="recoverWalletWithoutProfile_complete_continueButton">Continue</string>
+\n\n
```

Comment:
Is this on purpose?

Reply:
no, I will still add crowdin strings and make sure new lines are not present before merge